app/views/shared/_search_bar.html.erb

```diff
@@ -1,12 +1,17 @@
-<div class='c-input-group o-grid__item' data-controller='search' span@medium='2-6' span='3-5'>
-  <div class='c-input-group__icon'>
-    <%= icon_tag 'search', size: 'large' %>
-  </div>
-  <input id='js-search-input' class='c-input' data-search-target='input' data-action='keydown->search#query' data-test-id='search_input' value='<%= params[:query] %>' />
-  <div class='c-input-group__icon u-display-none' data-search-target='loader'>
-    <%= loader_tag size: 'small' %>
-  </div>
-</div>
+<%= form_tag search_path, method: :get, class: 'c-search c-input-group o-grid__item',
+            'span@medium': '2-6', span: '3-5',
+             data: { action: 'turbo:submit-start->search#loading turbo:submit-end->search#loading search#submit',
```

Comment:
We don't need to listen turbo:submit-start and turbo:submit-end method to change status of loading. Because we use submit function to change status to loading, after request finished, the page is reloaded, so the`is-loading` class is auto removed, and don't need to remove it again.